src/pytest_postgresql/__init__.py

```diff
@@ -19,8 +19,14 @@
 """Main module for pytest-postgresql."""
 
 import logging
+import sys
 
 __version__ = '1.2.0'
 
 
 logger = logging.getLogger(__name__)
+
+
+if sys.implementation.name == "pypy":
```

Comment:
This is only available at python 3.3 and forward. As long as I need that plugin at my work, I'd like to keep support for python 2.7 as well though. I'm trying something out at #69 (not pretty yet though)